pkg/neonvm/controllers/vm_controller.go

```diff
@@ -1378,6 +1416,19 @@ func podSpec(
 		},
 	}
 
+	pod.Spec.Containers[0].ReadinessProbe = &corev1.Probe{
```

Comment:
question: Any reason to add it here instead of as we construct the container above?

Reply:
and again, left over from the version when it was conditional

Reply:
Unresolving this - why keep it that way, then?

Reply:
oh, human factor mistake by me, accidentally did second round of code review fixes on top of wrong revision